code/style.css

```diff
@@ -1,7 +1,360 @@
+}
+
+.hero__topic {
+  position: absolute;
+  z-index: 1;
+  right: 300px;
+  top: 250px;
+  color: antiquewhite;
+  text-shadow: 2px 1px black;
+}
+
+.hero__video {
+  width: 100%;
+}
+
+.hero__video video {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+/* Main Content */
+
+.main-content {
+  display: flex;
+  flex-direction: row;
+  margin: 100px;
+}
+
+.main-content__section {
+  flex: 1;
+  padding: 50px;
+}
+
+/*  Signup form  */
+
+.signup-form {
+  background-color: yellow;
+  padding: 40px;
+  border-radius: 15px;
+  box-shadow: 0 16px 32px 0;
+  color: black;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+}
+
+.input-container__input:focus {
+  background-color: yellowgreen;
+}
+
+.input-container__input--full-width {
+  width: 100%;
+}
+
+.input-container--checkbox {
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+}
+
+.input-container--checkbox > label {
+  cursor: pointer;
+}
+
+.btn-submit {
+  width: 150px;
+  height: 200px;
+  position: relative;
+  animation-name: btn-animation;
+  animation-duration: 6s;
+  animation-direction: reverse;
+  animation-iteration-count: 2;
+}
+
+.submit {
+  background-color: green;
+  margin: 20px;
+  width: 100%;
+  border-radius: 20px;
+  box-shadow: 0 16px 32px 0;
+  cursor: grab;
+}
+
+.submit:hover {
+  box-shadow: none;
+}
+.submit:active {
+  box-shadow: none;
+}
+
+/* Animation */
+
+@keyframes btn-animation {
+  0% {
+    background-color: green;
+    transform: rotate(0deg);
+    left: 0px;
+  }
+  25% {
+    background-color: yellow;
+    transform: rotate(20deg);
+    left: 0px;
+  }
+  50% {
+    background-color: goldenrod;
+    transform: rotate(0deg);
+    left: 500px;
+  }
+  55% {
+    background-color: yellowgreen;
+    transform: rotate(0deg);
+    left: 500px;
+  }
+  75% {
+    background-color: greenyellow;
+    transform: rotate(0deg);
+    left: 500px;
+  }
+  100% {
+    background-color: green;
+    transform: rotate(-360deg);
+    left: 0px;
+  }
+}
+
+/* Accordion */
+
+.accordion-list {
+  background-color: violet;
+  padding: 40px;
+  border-radius: 15px;
+  box-shadow: 0 16px 32px 0;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  height: 100%;
+}
+.accordion-list__title {
+  text-align: center;
+}
+
+.accordion__question {
+  margin: 10px;
+  font-weight: bold;
+  font-size: 20px;
+  text-align: center;
+  border: 4px outset;
+  padding: 50px;
+  cursor: help;
+}
+
+.accordion__question:hover {
+  background-color: rgb(194, 94, 194);
+}
+.accordion__question > .icon-hand-down {
+  margin: 10px;
+  padding: 2px;
+}
+
+.accordion__question > .icon-hand-up {
+  margin: 10px;
+  padding: 2px;
+  display: none !important;
+}
+
+.accordion__question--active > .icon-hand-down {
+  display: none;
+}
+
+.accordion__question--active > .icon-hand-up {
+  display: unset !important;
+}
+
+.accordion__question--active + .accordion__answer {
+  height: 150px;
```

Comment:
Almost good, you have some small internal scroll over there in desktop view. Just increase the value a little bit ;)